zcash_client_sqlite/src/wallet/init.rs

```diff
@@ -19,6 +25,240 @@ use {
     zcash_primitives::legacy::keys::IncomingViewingKey,
 };
 
+struct WalletMigration0;
+
+migration!(
+    WalletMigration0,
+    "bc4f5e57-d600-4b6c-990f-b3538f0bfce1",
+    [],
+    "Initialize the wallet database."
+);
+
+impl RusqliteMigration for WalletMigration0 {
+    type Error = SqliteClientError;
+
+    fn up(&self, transaction: &Transaction) -> Result<(), SqliteClientError> {
+        transaction.execute_batch(
+            "CREATE TABLE IF NOT EXISTS accounts (
+                account INTEGER PRIMARY KEY,
+                extfvk TEXT NOT NULL,
+                address TEXT NOT NULL
+            );
+            CREATE TABLE IF NOT EXISTS blocks (
+                height INTEGER PRIMARY KEY,
```

Comment:
Since we have stronger types elsewhere in our codebase, do we ever consider constraints on some columns like height which could be confined to [0, UInt32.max]?

Reply:
If we want to do this, we should do so as a separate migration, for the same reason as is described in https://github.com/zcash/librustzcash/pull/600/files/8575d74dbb4af54cb45c1f1585f39c23754c3c8a#r946220645